### PR TITLE
Clean up ACM Ansible EE and add an automatic build process through GitHub actions

### PR DESCRIPTION
**Pull request by andreadecorte**
It will generate a new image on merge if any change in the relevant folder

### CODE REVIEW

File: tools/ansible-execution-environments/README.md [hunk 1]
```diff
@@ -1,18 +1,18 @@
 # ansible-execution-environments
+This folder contains an example of an Ansible Execution Environment built with Github Actions and pushed to Quay.
 
-```bash
-cd azure_ee_supported_1.13.0
-ansible-builder create
-cd context
-docker build -t quay.io/cdoan_rh/azure_ee_supported:1.13.0 .
-```
+It contains notably the stolostron.ocm collection to integrate Ansible and Red Hat Advanced Cluster Management and the Azure collection (which could be replaced with any other similar collection). Full list in [requirements.yml](ansible_ee_acm/requirements.yml)
```
[the remark this answers is not in the document]
Comment (sdminonne) [reply tools/ansible-execution-environments/README.md:4]: or rephrase at your wish

**Review by sdminonne — COMMENTED** [review]
Just 'nit for me but I would modify that comment. Thanks @andreadecorte

File: tools/ansible-execution-environments/README.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (andreadecorte) [reply tools/ansible-execution-environments/README.md]: fixed

**Review by sdminonne — APPROVED** [review]
:+1: lgtm thanks

**Pull request merged**